src/app/views/discover/games/view/overview/overview.ts

```diff
@@ -186,8 +189,16 @@ export default class RouteDiscoverGamesViewOverview extends BaseRouteComponent {
 
 	get routeTitle() {
 		if (this.game) {
-			const dev = this.game.developer;
-			return `${this.game.title} by ${dev.display_name} (@${dev.username})`;
+			let title = this.$gettextInterpolate('%{ gameTitle } by %{ user }', {
+				gameTitle: this.game.title,
+				user: this.game.developer.display_name,
+			});
+
+			if (this.browserBuilds.length) {
+				title += ' - ' + this.$gettext('Play Online');
```

Comment:
Maybe play online should be first? I think people won't see this if the name of the game is long, or the name of the user is long

Reply:
they dont need to see it on the tab - they need to see it in google results. it's close to the end and surrounded by hyphens so it does stand out enough. I think its fine.

Reply:
Yeah, but the title length is so short man....

Reply:
For the user it's better to put it at the end, for google it's better to put before... i really don't know

Reply:
I think either way this helps the dev get more traction from people searching to play a game without downloading it. people searching for the specific game to play it online will match this well, people looking to play just any game online will find our browser games list so i think we're good.